e2e/tests/nightly/admin/lock-workspace.spec.ts

```diff
@@ -0,0 +1,154 @@
+    // verify only the edit option is enabled on the snowmenu
+    await workspaceCard.verifyLockedWorkspaceMenuOptions();
+    await workspaceCard.clickLockedWorkspaceName(true);
+    const aboutPage = new WorkspaceAboutPage(page);
+    await aboutPage.waitForLoad();
+    // verify the banner includes the locking reason
+    const aboutLockReason = await aboutPage.extractReasonMessage();
+    expect(aboutLockReason).toEqual(reasonText);
+    // verify the lock icon is displaying for the locked workspace
+    const aboutLockedIcon = aboutPage.getAboutLockedWorkspaceIcon();
+    expect(aboutLockedIcon).toBeTruthy();
+
+    // verify DATA & ANALYSIS tabs are inactive for locked workspace
+    const analysisTab = await aboutPage.getTabState(Tabs.Analysis);
+    expect(analysisTab).toBe(false);
+    const dataTab = await aboutPage.getTabState(Tabs.Data);
+    expect(dataTab).toBe(false);
+
+    // verify share button is disabled
+    expect(await aboutPage.getShareButton().isCursorNotAllowed()).toBe(true);
+    await aboutPage.verifyLockedWorkspaceActionOptions();
+    // verify the user is able to navigate to edit page to update the description
+    await aboutPage.getAboutEditIcon();
+    const workspaceEdit = new WorkspaceEditPage(page);
+    await workspaceEdit.waitForLoad();
+    // verify user is able to edit the locked workspace
+    workspaceEditedName = await workspaceEdit.fillOutRequiredDuplicationFields();
+    const updateButton = workspaceEdit.getUpdateWorkspaceButton();
+    await updateButton.waitUntilEnabled();
+    await workspaceEdit.clickCreateFinishButton(updateButton);
+    await aboutPage.waitForLoad();
+  });
+
+  test('Verify the workspace is unlocked', async () => {
+    await navigation.navMenu(page, NavLink.WORKSPACE_ADMIN);
+    const workspaceAdminPage = new WorkspaceAdminPage(page);
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.getWorkspaceNamespaceInput().type(workspaceNamespace);
+    workspaceAdminPage.clickLoadWorkspaceButton();
+    await workspaceAdminPage.waitForLoad();
+    await workspaceAdminPage.clickUnlockWorkspaceButton(workspaceStatus.Unlock);
+    await workspaceAdminPage.waitForLoad();
+    expect(workspaceAdminPage.getLockWorkspaceButton(workspaceStatus.Lock));
+    await new HomePage(page).load();
+    const workspaceCard = await WorkspaceCard.findCard(page, workspaceEditedName);
+    expect(await workspaceCard.getWorkspaceLockedIcon()).toBeFalsy();
+    await workspaceCard.verifyWorkspaceCardMenuOptions();
+    await workspaceCard.clickWorkspaceName(true);
+    const dataPage = new WorkspaceDataPage(page);
+    await dataPage.waitForLoad();
+    // verify DATA & ANALYSIS tabs are aactive and accessible
+    const analysisPage = new WorkspaceAnalysisPage(page);
+    await openTab(page, Tabs.Analysis, analysisPage);
+    const aboutPage = new WorkspaceAboutPage(page);
+    await openTab(page, Tabs.About, aboutPage);
+    await aboutPage.waitForLoad();
```

Comment:
Check out the `openTab` function implementation. You will see that `await aboutPage.waitForLoad()` is not needed. Can you remove unneeded `waitForLoad` everywhere?